gettingstarted.html

```diff
@@ -0,0 +1,114 @@
+---
+layout: default
+title: Getting Started
+permalink: /gettingstarted/index.html
+---
+<h1>GETTING STARTED</h1>
+
+
+<h1>Using the Frame</h1>
+  <div class="grid">
+    <img class="col col-2-3" src="/assets/gettingstarted-1.png">
+    <p class="col col-1-3 pcenter">
+      Take the mouse to one of the corners, in order to reveal the frame.<br>
+      In order to change the settings and sensitivity of revealing the frame, kindly check out the next section.
+    </p>
+  </div>
+
+  <div class="grid">
+    <p class="col col-1-3">
+      Once the frame is revealed, you will see a lot of icons in it. Amongst the top, you will find the options to 
+      1. Go to Neighborhood View (F1)
+      2. Group View (F2)
+      3. Home (F3)
+      4. Activity (F4)
+      5. Journal (F5)
+      You can also find a lot of other options in the frame. You can explore these by clicking the respective icons.
+    </p>
+    <img class="col col-2-3" src="/assets/gettingstartedframe.png" padding=0> 
+  </div>
+
+<h1>Changing Frame Settings</h1>
+  <div class="grid">
+    <img class="col col-2-3" src="/assets/gettingstartedframe1.png">
+    <p class="col col-1-3 pcenter">
+      On home, click the XO Icon, and choose My Settings.<br>
+    </p>
+  </div>
+
+  <div class="grid">
+    <p class="col col-1-3">
+      In the Settings Window, select Frame Icon.
+    </p>
+    <img class="col col-2-3" src="/assets/gettingstartedframe2.png" padding=0> 
+  </div>
+
+  <div class="grid">
+    <img class="col col-2-3" src="/assets/gettingstartedframe3.png">
+    <p class="col col-1-3 pcenter">
+      In the Activation Delay, you can choose the delay it takes to show the frame from the edges as well as corners.
+    </p>
+  </div>
+
+<h1>Using Journal</h1>
+  <div class="grid">
+    <img class="col col-2-3" src="/assets/gettingstartedframejournal.png">
+    <p class="col col-1-3">
+      From the frame, Click on the Journal button.<br>
+    </p>
+  </div>
+  <div class="grid">
+    <p class="col col-1-3">
+      This will open up the Journal.<br>
+      You can think of this as call history, but a much more sophisticated, usable and powerful. It logs all the times you open an activity and what you do inside it. You can click on any instance and add/view/edit the description.
```

Comment:
Where do you click on the instances?  Please explain how you click the coloured icon to resume, and the ">" icon to view and edit the description, title, etc.